components/BacktoTopSection.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react'
+import Scroll from 'react-scroll'
+
+export default class TopArrowSection extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      atTopOfPage: true
+    }
+  }
+
+  componentDidMount () {
+    window.addEventListener('scroll', this.handleScroll)
+  }
+
+  componentWillUnmount () {
+    window.removeEventListener('scroll', this.handleScroll)
+  }
+
+  handleScroll = () => {
+    if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
```

Comment:
We can use the `window.scrollY` instead. It is supported in most modern browsers.